### PR TITLE
Complete the Cards

### PR DESCRIPTION
**Pull request by KBhautik6**
- [x] Checking the code is duplicate or not?
 - [x] Is these code has bug
 - [ ] Is code is check by ankit sir?
 - [ ] Are you going to merge this or not?

### CODE REVIEW

File: src/stories/Card/Card.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { StyledCardBox,StyledCardParagraph, StyledCardTitle, StyledOutlineButton, StyledBadgesText, StyledBadgesmallImage, StyledDissMissBadges, StyledBadgesMiddleText, StyledHorizontalCard, StyleContainer } from './Card.styled';
+
+export const Card = ({ title, content, variant, src, Badgetext, checkouttext, outofstock }) => {
```
Comment (tarakhant1297) [inline src/stories/Card/Card.js:4]: @KBhautik6 give proper name to card props

File: src/stories/Card/Card.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          <div>
+            <StyledCardTitle>{title}</StyledCardTitle>
+            <StyledCardParagraph>{content}</StyledCardParagraph>
+          </div>
+        </StyledCardBox>
+      </StyleContainer>
+    );
+  }
+
+  if (variant === "CardWithBadges") {
```
Comment (tarakhant1297) [inline src/stories/Card/Card.js:18]: @KBhautik6 change variant name for card

File: src/theme.js [hunk 3]
```diff
@@ -3,7 +3,11 @@ export const theme={
     secondary: "#4a4a68",
     success: "#07CB6E",
     warning: "#F3B664",
+<<<<<<< HEAD
```
Comment (tarakhant1297) [inline src/theme.js]: @KBhautik6 remove this merge conflict msg

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/theme.js (hunk 3 above)
Comment (KBhautik6) [reply src/theme.js]: okay

File: src/stories/Card/Card.js (hunk 1 above)
Comment (KBhautik6) [reply src/stories/Card/Card.js:4]: okay

File: src/stories/Card/Card.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    return (
+      <StyleContainer>
+        <StyledCardBox>
+          <StyledBadgesText>{Badgetext}</StyledBadgesText>
+          <StyledBadgesmallImage src={src} alt={alt} />
```
Comment (ankitadrixus08) [inline src/stories/Card/Card.js:23]: StyledBadgesmallImage >> StyledBadgeSmallImage

File: src/stories/Card/Card.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+      </StyleContainer>
+    );
+  }
+
+  if (variant === "CardsWithDissmiss") {
+    return (
+      <StyleContainer>
+        <StyledCardBox>
+          <StyledDissMissBadges>{Badgetext}</StyledDissMissBadges>
+          <StyledBadgesmallImage src={src} alt={alt} />
```
Comment (ankitadrixus08) [inline src/stories/Card/Card.js:38]: same as above StyledBadgeSmallImage

File: src/stories/Card/Card.styled.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
+// Card.styled.js
+import styled from 'styled-components';
+import theme from '../../theme'
+
+export const StyleContainer=styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 5%;
```
Comment (ankitadrixus08) [inline src/stories/Card/Card.styled.js:7]: here you give padding in % and below give padding in px please be consistence for whole project @KBhautik6

File: src/stories/Card/Card.styled.js [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+export const StyleContainer=styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 5%; 
+`
+
+export const StyledCardBox = styled.div`
+  display: flex;
+  flex-flow: column;
+  height: max-content;
```
Comment (ankitadrixus08) [inline src/stories/Card/Card.styled.js:13]: why use this @KBhautik6

File: src/stories/Card/Card.styled.js [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+  max-width: 19rem;
+  background-color: ${theme.white};
+  border-radius: 5px;
+  padding: 7px;
+  transition: 300ms ease;
```
Comment (ankitadrixus08) [inline src/stories/Card/Card.styled.js:18]: add one line here @KBhautik6

File: src/stories/Card/Card.styled.js [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+  font-weight:600;
+  padding:2px;
+`
+
+export const StyledDissMissBadges=styled(StyledBadgesText)`
+  top: 95px;
+  left: 630px;
+  padding: 8px 12px;
+  font-weight: 600;
+  &:hover,
```
Comment (ankitadrixus08) [inline src/stories/Card/Card.styled.js:56]: same for all when you use Pseudo classes please make proper indent and gapping @KBhautik6

File: src/stories/Card/Card.styled.js [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+  &:active {
+    box-shadow: 5px 4px 8px rgba(3, 0, 0, 0.1);
+  }
+`
+
+export const StyledBadgesMiddleText=styled(StyledBadgesText)`
+  top: 160px;
```
Comment (ankitadrixus08) [inline src/stories/Card/Card.styled.js:63]: why here top bottom without any position @KBhautik6

File: src/stories/Card/Card.styled.js [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+  &:active {
+    background-color: ${theme.primary};
+    color: ${theme.white};
+  }
+`
+
+export const StyledBadgesmallImage=styled(StyledCardBadges)`
+  width: 15%;
+  height: 25%;
+  padding-top: 2.1%;
```
Comment (ankitadrixus08) [inline src/stories/Card/Card.styled.js:93]: why 2.1% here @KBhautik6 please use round value also use shorthand property for this

File: src/stories/Card/Card.styled.js [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+  font-weight: 600;
+  &:hover,
+  &:active {
+    box-shadow: 5px 4px 8px rgba(3, 0, 0, 0.1);
+  }
+`
+
+export const StyledBadgesMiddleText=styled(StyledBadgesText)`
+  top: 160px;
+  left: 470px;
```
Comment (ankitadrixus08) [inline src/stories/Card/Card.styled.js:64]: why left: 470px here @KBhautik6

File: src/stories/Card/Card.styled.js [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+  height: 25%;
+  padding-top: 2.1%;
+`
+
+export const StyledHorizontalCard=styled.div`
+  display: flex;
+  gap: 5px;
+  flex-flow: row;
+  height: max-content;
+  max-width: 720px;
```
Comment (ankitadrixus08) [inline src/stories/Card/Card.styled.js:101]: when you use max-width than use width: 100% after that is compulsory @KBhautik6

**Pull request merged**